hbw/config/initializers/hbw.rb

```diff
@@ -7,4 +7,6 @@
   validated_config = HBW::ConfigValidator.(Settings::HBW.deep_symbolize_keys)
 
   HBW::Widget.config = HBW::Config.new(validated_config.to_h)[:hbw]
+
+  Rails.application.config.action_cable.mount_path = '/widget/cable'
```

Comment:
`mount_path` or `url`? I think `url` semantically more appropriate, because we don't mount cable as separate service.